### ISSUE TITLE
Can't build Parse SDK iOS 

### ISSUE THREAD

**Issue by gonchar** [ParsePlatform/Parse-SDK-iOS-OSX#187]
Hello! I can't build Parse SDK. It fails here
```
▸ Running script 'Generate Localizable Strings'
** BUILD FAILED **

The following build commands failed:
	PhaseScriptExecution Generate\ Localizable\ Strings /var/root/Library/Developer/Xcode/DerivedData/Parse-emosnuyslnnxltdskewkkizbnaiv/Build/Intermediates/Parse.build/Release-iphoneos/Parse-iOS.build/Script-81C3823C19CCA9950066284A.sh
(1 failure)
Failed to build framework for iphoneos.
```

Full terminal log: https://gist.github.com/gonchar/cbc573202ccc225d0c81

**Comment by nlutsenko** [comment]
Hey @gonchar, thanks for reporting.
It looks like you are trying to execute this via `sudo` which makes `xcodebuild` run as sudo, which can cause really terrible things, as well as it's an unsupported use case.
Any chance you can try it without `sudo` aka simply running `rake package:frameworks`?
I am also wondering of the initial reason to use sudo for this, since running without sudo works even on fresh install of OS X.

**Comment by gonchar** [comment]
It works the same without sudo. Here is the terminal log: https://gist.github.com/gonchar/7bfd0125177e9476fcc3
btw, I'm on El Capitan OSX

**Comment by nlutsenko** [comment]
I suspect that you are running in the problem fixed in #181. 
Any chance you can try that branch and let us know if that's the case?

(the problem arises from missing he strings in Xcode 6.4 running El Capitan)

**Comment by gonchar** [comment]
thank you! now it works perfectly.
